sandboxed_containers/understanding-sandboxed-containers.adoc

```diff
@@ -0,0 +1,29 @@
+[id="understanding-sandboxed-containers"]
+= Understanding OpenShift sandboxed containers
```

Comment:
You can use your `OpenShift sandboxed containers` attribute here

Reply:
PV2/FCC Compliance - not to use attributes in the title